tools/github-action/go.mod

```diff
@@ -7,19 +7,22 @@ require (
 	github.com/sirupsen/logrus v1.9.3
 	github.com/spf13/cobra v1.7.0
 	k8s.io/api v0.27.3
-	k8s.io/apimachinery v0.27.3
+	k8s.io/apimachinery v0.27.4
 	kraftkit.sh v0.6.4-108-g28b7051
 )
 
+replace kraftkit.sh => ../..
```

Comment:
This did not work for me, why shouldn't we just set a version and bump through dependabot?